livre/lib/cache/HistogramObject.cpp

```diff
@@ -31,77 +30,79 @@
 
 namespace livre
 {
+namespace
+{
+template< class SRC_TYPE >
+void binData( const SRC_TYPE* rawData,
```

Comment:
start function with an action verb, e.g. fillHistogram, and pass the histogram object to it instead of the splitting dstData and its size as an obscure "resolution" param?